var/spack/repos/builtin/packages/libzip/package.py

```diff
@@ -46,14 +36,32 @@ def url_for_version(self, version):
 
     depends_on("zlib-api")
 
-    # Build system
-    build_system(
-        conditional("cmake", when="@1.4:"), conditional("autotools", when="@:1.3"), default="cmake"
-    )
+    variant("gnutls", default=True, description="Enable gnutls support")
+    variant("bzip2", default=True, description="Enable bzip2 support")
+    variant("lzma", default=True, description="Enable lzma support")
+    variant("openssl", default=True, description="Enable openssl support")
+    variant("zstd", default=True, description="Enable zstd support", when="@1.8:")
+    variant("mbedtls", default=True, description="Enable mbedtls support")
+    depends_on("gnutls", when="+gnutls")
+    depends_on("bzip2", when="+bzip2")
+    depends_on("lzma", when="+lzma")
+    depends_on("openssl", when="+openssl")
+    depends_on("mbedtls", when="+mbedtls")
+    depends_on("zstd", when="+zstd")
 
     @property
     def headers(self):
         # Up to version 1.3.0 zipconf.h was installed outside of self.prefix.include
         return find_all_headers(
             self.prefix if self.spec.satisfies("@:1.3.0") else self.prefix.include
         )
```

Comment:
This is now always false since all versions pre-1.3.0 are removed.
```suggestion
    @property
    def headers(self):
        return find_all_headers(self.prefix.include)
```
but then I think this is essentially the default implementation (`_headers_default_handler`, where `find_all_headers` is recursive) and doesn't need to be implemented here. So:
```suggestion
```